web/src/app/alerts/components/CreateAlertSpeedDial.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react'
+import { KeyChange as ServicesIcon } from 'mdi-material-ui'
+import { Notifications as AlertsIcon } from '@material-ui/icons/'
+import classnames from 'classnames'
+import AlertForm from './AlertForm'
+import CreateAlertDialog from '../CreateAlertDialog'
+import withStyles from '@material-ui/core/styles/withStyles'
+import SpeedDial from '../../util/SpeedDial'
+
+const styles = theme => ({
+  speedDial: {
+    position: 'fixed',
+    bottom: '2em',
+    right: '2em',
+    zIndex: 9001,
+  },
+  transitionUp: {
+    transform: 'translate3d(0, -62px, 0)',
+    transition: theme.transitions.create('transform', {
+      duration: theme.transitions.duration.enteringScreen,
+      easing: theme.transitions.easing.easeOut,
+    }),
+  },
+  warningTransitionUp: {
+    transform: 'translate3d(0, -7.75em, 0)',
+    transition: theme.transitions.create('transform', {
+      duration: theme.transitions.duration.enteringScreen,
+      easing: theme.transitions.easing.easeOut,
+    }),
+  },
+  speedDialClose: {
+    transform: 'translate3d(0, 0, 0)',
+    transition: theme.transitions.create('transform', {
+      duration: theme.transitions.duration.leavingScreen,
+      easing: theme.transitions.easing.sharp,
+    }),
+  },
```

Comment:
Are these transitions required? We are getting the same transitions on other speed dials without setting any additional CSS